security/medusa/l2/acctype_syscall.c

```diff
@@ -69,12 +69,7 @@ medusa_answer_t asmlinkage medusa_syscall_i386(
 		process_kern2kobj(&proc, current);
 		retval = MED_DECIDE(syscall_access, &access, &proc, &proc);
 	}
-	/* this needs more optimization some day */
-	if (retval == MED_NO)
-			return 0; /* deny */
-	if (retval != MED_SKIP)
-		return 1; /* allow */
-	return 2; /* skip trace code */
+	return retval;
```

Comment:
1) acctype syscall is not used (for now) and cannot be used via LSM :(
2) I mean, the purpose of this snippet of code is conversion from medusa codes to kernel ones in the place where medusa_syscall_i386 was originally used (in the form of some L1 patch in kernel)
3) I suggest to left it as was

Reply:
Well it seemed to me as some old piece of code since the comments do not reflect current reality.

0 != DENY
1 != ALLOW